JSONCodable/JSONDecodable.swift

```diff
@@ -74,6 +76,26 @@ public extension Array where Element: JSONDecodable {
     }
 }
 
+
+
+
+private extension String {
```

Comment:
prefer as a private function inside of JSONDecoder since we don't use this anywhere else